dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/URAPIs_TDSLDAP_SearchBaseTest.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.security.wim.adapter.ldap.fat;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.security.registry.test.UserRegistryServletConnection;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+import componenttest.topology.utils.LDAPUtils;
+
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class URAPIs_TDSLDAP_SearchBaseTest {
```

Comment:
I am not a fan of this name. Are we just testing whether operational group member attributes are filtered on search base? The URAPI tests probably shouldn't even be in LDAP fat. Those are the UserRegistry API tests, which probably belong in com.ibm.ws.security.wim.registry_fat instead. Additionally, does it need TDS in it?

Reply:
Rename the test server accordingly as well.

Reply:
Also I am not really seeing a test that tests the change directly? Most of these tests should not be relevant, no? We also probably need to make sure that these tests only run with the real LDAP servers. I doubt that UnboundID nor ApacheDS support operational group member attributes?

Reply:
I would like to see a test that explains that a group that is not under the group base entry that it is now passing (and fails prior to your change).

Might be good to test with just the ldapRegistry baseDn and also with base entries under the group. not sure if that is possible given the entries in our TDS server.